app/services/users/exception_handler.rb

```diff
@@ -11,15 +11,15 @@ class ExceptionHandler
     attr_reader :error, :service
 
     # @param error [ErrorClass] An external service error
-    # @param service [String] The name of the external service (i.e. 'Vet360', 'MVI', 'EMIS')
+    # @param service [String] The name of the external service (i.e. 'Vet360', 'MVI', 'VA Profile')
     #
     def initialize(error, service)
       @error = validate!(error)
       @service = service
     end
 
     # Serializes the initialized error into one of the predetermined error types.
-    # Uses error classes that can be triggered by MVI, EMIS, or Vet360.
+    # Uses error classes that can be triggered by MVI, VA Profile, or Vet360 (aka VA Profile).
```

Comment:
🤔 I may be missing context here, but this reads effectively like "can be triggered by... VA Profile... or... VA Profile" to me? Took me a moment to check back, thinking "wait, isn't that what it just said before? ... yes it is, huh".

Would something like
```
# Uses error classes that can be triggered by MVI, or VA Profile (aka Vet360).
```
carry the same meaning, or is there something else going on here?

Reply:
Yeah, good point @Mitch-A6. Changed. Thanks!